app/models/standard.rb

```diff
@@ -0,0 +1,4 @@
+class Standard < ApplicationRecord
+    has_many :subjects 
```

Comment:
@rubyboy-hub  The ER dia does not show this association. Do you still want to use it?

Reply:
yes, i had insert an standard foreign key to subjects, so all standard wise subject can be inserted to subject table. When a user class related to a standard. All his/her subject can be seen using this.